### PR TITLE
Update collabora.sh

### PR DESCRIPTION
**Pull request by ezraholm50**
After #40 I've polished the script a bit and added some code, view the # out commented lines for info.

### CODE REVIEW

File: static/collabora.sh [hunk 1]
```diff
@@ -70,9 +109,9 @@ else
 
   # SSL configuration, you may want to take the easy route instead and use Lets Encrypt!
   SSLEngine on
-  SSLCertificateChainFile $CERTFILES/$DOMAIN/chain.pem
-  SSLCertificateFile $CERTFILES/$DOMAIN/cert.pem
-  SSLCertificateKeyFile $CERTFILES/$DOMAIN/privkey.pem
+  SSLCertificateChainFile $CERTFILES/$CLEANDOMAIN/chain.pem
```
Comment (ezraholm50) [inline static/collabora.sh]: Had the wrong domain

File: static/collabora.sh [hunk 2]
```diff
@@ -1,15 +1,56 @@
...
+# Letsencrypt
 LETSENCRYPTPATH=/etc/letsencrypt
 CERTFILES=$LETSENCRYPTPATH/live
+# WANIP
+WANIP4=$(dig +short myip.opendns.com @resolver1.opendns.com)
+# Misc
 SCRIPTS=/var/scripts
 
-# Message
-whiptail --msgbox "Please before you start make sure port 443 is directly forwarded to this machine or open!" 20 60 2
```
Comment (ezraholm50) [inline static/collabora.sh]: Added nmap 443 port check on WAN ipv4 address

File: static/collabora.sh [hunk 3]
```diff
@@ -1,15 +1,56 @@
 #!/bin/bash
-DOMAIN=$(whiptail --title "Techandme.se Collabora" --inputbox "Nextcloud url, make sure it looks like this: cloud\.yourdomain\.com" 10 60 cloud\.yourdomain\.com 3>&1 1>&2 2>&3)
```
Comment (ezraholm50) [inline static/collabora.sh]: Updated the URL with 2 backslashes (nextcloud collabora docs)

**Comment by enoch85** [comment]
I'll just merge this. Will redo everything anyway. :)

**Pull request merged**